include/seqan3/io/sequence_file/format_genbank.hpp

```diff
@@ -0,0 +1,252 @@
+    sequence_file_format_genbank & operator=(sequence_file_format_genbank const &) = delete;
+    //!\brief Move construction is defaulted.
+    sequence_file_format_genbank(sequence_file_format_genbank &&) = default;
+    //!\brief Move assignment is defaulted.
+    sequence_file_format_genbank & operator=(sequence_file_format_genbank &&) = default;
+    //!\}
+
+    //!\brief The valid file extensions for this format; note that you can modify this value.
+    static inline std::vector<std::string> file_extensions
+    {
+        { "genbank" },
+        { "gb" },
+        { "gbk" },
+    };
+
+    //!\copydoc SequenceFileInputFormat::read
+    template <typename stream_type,     // constraints checked by file
+              typename seq_legal_alph_type, bool seq_qual_combined,
+              typename seq_type,        // other constraints checked inside function
+              typename id_type,
+              typename qual_type>
+    void read(stream_type                                                            	& stream,
+              sequence_file_input_options<seq_legal_alph_type, seq_qual_combined> const & options,
+              seq_type                                                                  & sequence,
+              id_type                                                                   & id,
+              qual_type                                                                 & SEQAN3_DOXYGEN_ONLY(qualities))
+    {
+        using stream_char_t = typename stream_type::char_type;
+        auto stream_view = std::ranges::subrange<decltype(std::istreambuf_iterator<stream_char_t>{stream}),
+                                                 decltype(std::istreambuf_iterator<stream_char_t>{})>
+                           {std::istreambuf_iterator<stream_char_t>{stream},
+                            std::istreambuf_iterator<stream_char_t>{}};
+        auto stream_it = std::ranges::begin(stream_view);
+
+        if (!(std::ranges::equal(stream_view | view::take_until_or_throw(is_cntrl || is_blank), std::string{"LOCUS"})))
+            throw parse_error{"An entry has to start with the code word LOCUS."};
+
+        //ID
+        if constexpr (!detail::decays_to_ignore_v<id_type>)
+        {
+            if (options.embl_genbank_complete_header)
+            {
+                std::ranges::copy(std::string{"LOCUS"}, std::back_inserter(id));
+
+                while (!(std::ranges::equal(stream_view | view::take_until_or_throw(is_cntrl || is_blank),
+                                            std::string{"ORIGIN"})))
+                {
+                        std::ranges::copy(stream_view | view::take_line_or_throw
+                                                	  | view::char_to<value_type_t<id_type>>,
+                                     	  std::back_inserter(id));
+                        std::ranges::copy(std::string{"\n"}, std::back_inserter(id));
```

Comment:
```suggestion
                        id.push_back('\n');
```
You are using the std::back_inserter which delegates to push_back anyway.